demo/boundingbox/dist/index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>DeepDetect Demo : BoundingBox</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+</head>
+<body>
+
+  <div class="container">
+
+  <div class="boundingBox" data-service="faces" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
+  <hr/>
+  <div class="boundingBox" data-service="face_gender" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
+  <hr/>
+  <div class="boundingBox" data-service="clothes"></div>
+  <hr/>
+  <div class="boundingBox" data-service="generic"></div>
+
+</div>
+
+  <script>__REACT_DEVTOOLS_GLOBAL_HOOK__ = parent.__REACT_DEVTOOLS_GLOBAL_HOOK__</script>
+  <script type="text/javascript" src="/app.js"></script>
```

Comment:
In my testing setup I had to replace with `src="assets/app.js`, is this a bug or a wrong setup from my part ?